lib/src/source/git.dart

```diff
@@ -426,6 +433,20 @@ class BoundGitSource extends CachedSource {
     _updatedRepos.add(path);
   }
 
+  ///  Clean-up [dirPath] if it's an invalid git repository
+  void _cleanInvalidGitRepoCache(String dirPath) {
+    final Directory directory = Directory(dirPath);
```

Comment:
What do you need this for anymore?

I'm just wondering if it makes if we pass `directory.path` in `runProcessSync`?